### ISSUE TITLE
1.8.1 Release - defineNuxtPlugin not defined

### ISSUE THREAD

**Issue by travis-r6s** [gbicou/nuxt-urql#147]
The latest release `v1.8.1` throws this error when using it: `defineNuxtPlugin is not defined`
```
at ./node_modules/.pnpm/@bicou+nuxt-urql@1.8.1_defu@6.1.4_graphql@16.8.1_rollup@4.17.2_vue@3.4.27_typescript@5.4.5_/node_modules/@bicou/nuxt-urql/dist/runtime/plugin.mjs:9:1
```

~~When looking at that file, the function is indeed exported, but not defined. Don't know if it just needs to be rebuilt after the Nuxt package upgrades in this release, or if the import needs to be added manually now?~~

It seems that import (and some others) were removed in this commit: https://github.com/gbicou/nuxt-urql/commit/1f4755c7878b8c4f6f36ce86c345265f68e23293#diff-7f98c423852d4ee9e5105730816a219164437326eca7dd00d865addf4b0ed8d0L2

**Comment by travis-r6s** [comment]
If it just needs to be added manually, I can submit a PR for this change!

**Comment by gbicou** [comment]
fixed in 1.8.2

**Comment by travis-r6s** [comment]
@gbicou Great, thanks for the quick fix! 🙌
